frontend/src/shared/util/permission-utils.ts

```diff
@@ -34,3 +34,11 @@ export const isAdminOrProjectOwner = (user: IUser, projectPermissions: Permissio
         hasPermission(Permission.ADMIN, user.permissions)
     );
 };
+
+export const projectCreationAdminRequired = (isAdminOnly: boolean, user: IUser) => {
```

Comment:
Again not sold on my suggestion but I think the existing name is a bit confusing especially if we expand things based on additional groups/roles which is on the roadmap.
```suggestion
export const enableProjectCreation = (isAdminOnly: boolean, user: IUser) => {
```

Reply:
Good point, this wouldn't make sense if we're integrating user groups in this function in the future